install.sh

```diff
@@ -64,8 +64,10 @@ get_binaries() {
   case "$PLATFORM" in
     darwin/386) BINARIES="kuber" ;;
```

Comment:
Does this actually need to be removed? I don't find the asset: 

<img width="417" alt="Screenshot 2022-10-04 at 14 49 01" src="https://user-images.githubusercontent.com/8149019/193823384-07d840df-eeae-4666-9fee-2b0bee6009f5.png">